CMSgov/mint-app
# Easi 2612/user account migration plan ops eval and learning

# EASI-2612

## Changes and Description

- Ops Eval and Learning updated to use the user table
- Auditing config updated
- Operational Needs trigger disabled for this task list section


## How to test this change
1. Bring the app up, and seed data.
2.  Verify that you can update Ops Eval and Learning Section
3. Verify Unit test pass as expected. 


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
